src/test/java/org/opensearch/plugin/insights/core/service/categorizor/SearchQueryCategorizerTests.java

```diff
@@ -114,7 +114,7 @@ public void testAggregationsQuery() {
         verify(searchQueryCategorizer.getSearchQueryCounters().getAggCounter()).add(valueCaptor.capture(), tagsCaptor.capture());
 
         double actualValue = valueCaptor.getValue();
-        String actualTag = (String) tagsCaptor.getValue().getTagsMap().get("type");
+        String actualTag = (String) tagsCaptor.getValue().getTagsMap().get("agg_type");
```

Comment:
nit: Why not use the `AGGREGATION_TYPE_TAG`?

Reply:
Will have to make it public

Reply:
No that is because of the typo in package name. Don't need public if they are in same package!